package.json

```diff
@@ -15,16 +15,18 @@
   ],
   "scripts": {
     "lint": "eslint .",
-    "test": "echo 'TODO add jest as a seperate piece of work'",
+    "test": "jest --silent --runInBand",
```

Comment:
nitpick: Just this caught my eye because in the documentation is said the tests run in parallel.
`--runInBand` is safer but also takes longer and makes the tests to run sequentially.

Reply:
Good spot, I'll remove 👍